### DISCUSSION TITLE
Copilot for PRs not working in our repos

### DISCUSSION THREAD

**Discussion by adam-savard** [community/community#73598] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi! I enrolled my organization's repos in copilot for PRs last week, and the repo owner just approved it to be used. We've reinstalled it as well, but typing the relevant flag doesn't seem to do anything.

**Comment by MattSPalmer** [comment]
We are seeing the same behavior. The app is enabled on the repo, it has R/W access to PRs, and nothing is happening when we either create or update PR descriptions and/or comments to contain the form `copilot:keyword`.

**Comment by zachintosh** [comment]
We are seeing the same issue as well. Was it ever resolved?
